gradle-server/src/main/java/com/github/badsyntax/gradle/utils/Utils.java

```diff
@@ -18,7 +18,11 @@ public static boolean hasCompatibilityError(Version gradleVersion, Version javaV
 
 	private static Version getLowestGradleVersion(Version javaVersion) {
 		// Ref: https://docs.gradle.org/current/userguide/compatibility.html
-		if (javaVersion.isAtLeast("19")) {
+		if (javaVersion.isAtLeast("21")) {
+			return new Version("8.4-rc-1");
```

Comment:
Let's simply use `8.4` according to https://docs.gradle.org/current/userguide/compatibility.html

And also consider adding a TODO marker here since 8.4 is now partial support according to the matrix. (`Support for running Gradle` is `N/A` now)

Reply:
Cool `v8.4` is out now! Sounds good, I'll update the version and add a TODO comment 🙂